modules/large-deployments/pages/multi-server.adoc

```diff
@@ -5,6 +5,6 @@ If you need to manage a large number of clients, in most cases you can do so wit
 However, if you need to manage tens of thousands of clients, you might find it easier to use multiple {productname} Servers, in a hub, to manage them.
 
 {productname} Hub is a collection of components to help you manage very large deployments.
```

Comment:
```suggestion
{productname} Hub is a collection of components to help you manage very large deployments.
```

Hub is (currently) just one component: the XMLRPC API. Going forward more will be added, but we are not there yet. For now I think we should just call it "a topology".

Reply:
I think it's OK at least for the time being. Thanks.